tailcall/tailcall.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+wget https://github.com/tailcallhq/tailcall/releases/download/v0.13.0/tailcall-x86_64-unknown-linux-gnu
```

Comment:
using hardcoded version is not recommended. Rather use following:
`
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
brew tap tailcallhq/tailcall
brew install tailcall
tc ./hello.graphql
`

Reply:
Thanks, modified it!